src/Widget/WidgetType/SearchResults.php

```diff
@@ -384,14 +388,14 @@ class SearchResults extends WidgetTypeBase
     /**
      * SearchResults constructor.
      *
-     * @param array $pluginDefinition
-     * @param array $configuration
-     * @param bool $cleanup
+     * @param array             $pluginDefinition
+     * @param array             $configuration
+     * @param bool              $cleanup
```

Comment:
I don't think this is part of the code style enforcement in this repository. Was this your editor making these changes automatically?

Reply:
jup editor did this...